steps/create-apple-s-documentation/1.0.0/step.yml

```diff
@@ -0,0 +1,132 @@
+title: Create Apple's documentation
+summary: |
+  This step helps to create a HTML representation of the Framework documentation, using Jazzy as engine
+description: |
+  This step helps to create a HTML representation of the Framework documentation, using Jazzy as engine.
+  Objective-C and Swift are e supported, but there is no declared supported for mixed language projects.
+  THis was created with the goal to export as a reach HTML documents for Frameworks.
+website: https://github.com/FutureWorkshops/bitrise-step-create-apple-s-documentation
+source_code_url: https://github.com/FutureWorkshops/bitrise-step-create-apple-s-documentation
+support_url: https://github.com/FutureWorkshops/bitrise-step-create-apple-s-documentation/issues
+published_at: 2019-11-22T20:02:44.869801+01:00
+source:
+  git: https://github.com/FutureWorkshops/bitrise-step-create-apple-s-documentation.git
+  commit: 073e907bb4d64cf5178adb39e75cea5c5db42f49
+host_os_tags:
+- osx-10.10
+project_type_tags:
+- ios
+- macos
+type_tags:
+- utility
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  check_only:
+  - name: xcode
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- opts:
+    is_expand: true
+    is_required: true
+    summary: Path in where the script will be run
+    title: Project path
+  project_path: .
+- language: swift
+  opts:
+    description: |
+      Language used in the project. Either 'swift', or 'objc'.
+
+      At the moment, the step do not support the generation, of documentation
+      for project with mixed language, even if it is a supported feature on Jazzy.
+    is_expand: true
+    is_required: true
+    summary: Language used in the project.
+    title: Language
+    value_options:
+    - objc
+    - swift
+- author: null
+  opts:
+    description: ""
+    is_expand: true
+    is_required: true
+    summary: Author of the Framework
+    title: Author
+- opts:
+    is_expand: true
+    is_required: true
+    summary: SDK to which the code is directed
+    title: SDK
+    value_options:
+    - iphone
+    - mac
+    - tv
+    - none
+  sdk: iphone
+- opts:
+    description: ""
+    is_expand: true
+    is_required: true
+    summary: Version string that will be used as Module Version in the documentation
+    title: Module Version
+  version: 1.0.0
+- framework_root: .
+  opts:
+    is_expand: true
+    is_required: true
+    summary: Root path of where the Framework is
+    title: Root path of the Framework
+- module: null
+  opts:
+    is_expand: true
+    is_required: true
+    summary: Name of the module (also used as scheme)
+    title: Module name
+- acl: public
+  opts:
+    is_expand: true
+    is_required: true
+    title: The minimun ACL used to list the included classes on the doc set
+    value_options:
+    - open
+    - public
+    - internal
+    - private
+- opts:
```

Comment:
It may be useful to add an output with the generated docs path, so it can be used in later steps easily. In this case the output path can even be a temporary directory generated by the step.